certified-connectors/TOPdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,11510 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "A Connector for TOPdesk, which entails connectors for Supporting Files, and Incident Management. All endpoint for creating, processing and closing incidents are available in the connector, aswell as alle endpoints for creating and handling branches, locations, persons and operators.",
+    "version": "1.0",
+    "title": "TOPdesk API V1",
```

Comment:
@CJusjong Please make the title to meet certification [standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title). 

1) Remove capitalization after first letter
2) Remove `API` and `V1` words